### PR TITLE
lopper: assists: xlnx_overlay_dt: Migrate the DT overlays to sugar sy…

### PR DESCRIPTION
**Pull request by NavaKishoreManne**
…ntax

Migrate the DT overlays to sugar syntax instead of hand coding overlay fragments structure.

### CODE REVIEW

File: lopper/assists/xlnx_overlay_dt.py
Comment (kedareswararao) [inline lopper/assists/xlnx_overlay_dt.py:202]: unneeded print

File: lopper/assists/xlnx_overlay_dt.py
Comment (kedareswararao) [inline lopper/assists/xlnx_overlay_dt.py:204]: same here

**Comment by zeddii** [comment]
I'll wait for fixes based on the review comments before merging.

**Review by kedareswararao — APPROVED** [review]
Changes looks fine to me

**Pull request merged**
